src/inter_dc_communication_sender.erl

```diff
@@ -132,7 +125,7 @@ stop(timeout, State=#state{socket=Socket}) ->
 
 stop_error(timeout, State=#state{socket=Socket}) ->
     _ = gen_tcp:close(Socket),
-    {stop, error, State}.
+    {stop, normal, State}.
```

Comment:
This will cause the process to not be restarted in the event of a timeout, is that correct behavior?